core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -350,6 +350,18 @@
   MASTER_METADATA_SUSPENDABLE("master.metadata.suspendable", "false", PropertyType.BOOLEAN,
       "Allow tablets for the " + MetadataTable.NAME
           + " table to be suspended via table.suspend.duration."),
+  MASTER_STARTUP_TSERVER_AVAIL_MIN_COUNT("master.startup.tserver.avail.min.count", "0",
+      PropertyType.COUNT,
+      "Minimum number of tservers that need to be registered before master will "
```

Comment:
Should document the behavior of a zero value.  Could document the version in which this option was added.